Rakefile

```diff
@@ -4,9 +4,16 @@ require "bundler/gem_tasks"
 require "rake/testtask"
 
 Rake::TestTask.new(:test) do |t|
-  t.libs << "test"
-  t.libs << "lib"
```

Comment:
`lib` is added by default: https://ruby-doc.org/stdlib-3.0.2/libdoc/rake/rdoc/Rake/TestTask.html, seems like we don't need `test`